build-info-extractor/src/main/java/org/jfrog/build/extractor/clientConfiguration/ArtifactoryClientConfiguration.java

```diff
@@ -454,6 +455,10 @@ public Boolean isPublishArtifacts() {
             return getBooleanValue(PUBLISH_ARTIFACTS, true);
         }
 
+        public Boolean generateDeployArtifactsMap() {
```

Comment:
Maybe better to rename generateDeployArtifactsMap to shouldAddDeployableArtifacts ?